models/taskRequests.js

```diff
@@ -59,11 +59,12 @@ const fetchTaskRequestById = async (taskRequestId) => {
     if (taskRequestData) {
       taskRequestData.id = taskRequestSnapshot.id;
       taskRequestData.url = new URL(`/taskRequests/${taskRequestData.id}`, config.get("services.rdsUi.baseUrl"));
+
+      return {
+        taskRequestData,
+        taskRequestExists: true,
+      };
```

Comment:
why are we making this changes in the tests PR?
This pr should only include tests

Reply:
This was a small fix and since this is a stacked PR to #1276 , I must have mixed the changes.